karma.conf.js

```diff
@@ -26,6 +26,7 @@ module.exports = function(config) {
     // list of files / patterns to load in the browser
     files: [
       'lib/jquery.min.js',
+      'src/common/lib/xmodule/xmodule/js/common_static/js/src/accessibility_tools.js',
```

Comment:
This won't work-- this file lives in edx-platform. The lms and studio directories are subdirectories within ora itself.

The code either needs to be robust to that global function not existing, or we need to create our own sr div.